onnxruntime/core/providers/nnapi/nnapi_builtin/nnapi_execution_provider.cc

```diff
@@ -117,22 +117,45 @@ NnapiExecutionProvider::GetCapability(const onnxruntime::GraphViewer& graph_view
     }
   }
 
+  // Get all the NodeUnits in the graph_viewer
+  std::vector<std::unique_ptr<NodeUnit>> node_unit_holder;
+  std::unordered_map<const Node*, const NodeUnit*> node_unit_map;
+
+  std::tie(node_unit_holder, node_unit_map) = GetAllNodeUnits(graph_viewer);
```

Comment:
>std::tie(node_unit_holder, node_unit_map) [](http://example.com/codeflow?start=2&length=41)

fyi, could also use structured bindings:
`const auto [node_unit_holder, node_unit_map] = GetAllNodeUnits(graph_viewer);`

Reply:
Unfortunately, the node_unit_map need to be captured by the lambda below, seems lambda cannot capture structurally bound reference unless we copy it.

Reply:
Unfortunately, `node_unit_map` need to be captured by lambda below, seems lambda cannot bind structurally bound reference unless we copy it